apps/api/src/api/application-api.ts

```diff
@@ -102,7 +103,12 @@ export const getApplicationById = async ({ applicationId }: { applicationId: num
 	const database = getDbInstance();
 	const applicationRepo: ApplicationService = applicationService(database);
 
-	const result = await applicationRepo.getApplicationById({ id: applicationId });
```

Comment:
Just changing the service method used will accomplish the goals of the ticket, without affecting existing usage of getApplicationById